packages/generator/ni_measurement_plugin_sdk_generator/client/_support.py

```diff
@@ -123,6 +123,11 @@ def get_configuration_and_output_metadata_by_index(
     """Returns the configuration and output metadata of the measurement."""
     configuration_parameter_list = []
     for configuration in metadata.measurement_signature.configuration_parameters:
+        if configuration.message_type:
+            raise click.ClickException(
+                "Measurement configurations with message datatype are not supported."
```

Comment:
Include bad message_type like below.
f"Measurement configurations do not support message data types ({configuration.message_type} is unsupported)."

Reply:
I have updated the error messages.